nodecg-io-twitter/extension/index.ts

```diff
@@ -0,0 +1,83 @@
+import { NodeCG } from "nodecg/types/server";
+import { NodeCGIOCore } from "nodecg-io-core/extension";
+import { Service, ServiceProvider } from "nodecg-io-core/extension/types";
+import { emptySuccess, success, error, Result } from "nodecg-io-core/extension/utils/result";
+const Twitter = require("twitter"); // eslint-disable-line @typescript-eslint/no-var-requires
```

Comment:
This should be an actual import:
```suggestion
import Twitter = require("twitter");
```
This also explains why I removed the `typeof` in https://github.com/codeoverflow-org/nodecg-io/pull/33/files#r447090680. Using an actual import results in types being loaded correctly and therefore fixes the issue about it being a function which allows of getting rid of the `typeof`. I've forgotten to add this comment, sorry.

Reply:
Also I couldn't get VS Code autocompletion working with const and the typeof and this fixed it.

Reply:
I am pretty sure I tried this one and it didn't work, but maybe i messed something else up at this time.
But yes it works now...